pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetOptionsForActivityInstanceQuestionRoute.java

```diff
@@ -131,6 +131,9 @@ public RemoteAutoCompleteResponse handle(Request request, Response response) {
         List<PicklistOption> optionMatches = options.stream().filter(option -> option.getOptionLabel().toUpperCase()
                 .contains(autoCompleteQuery.toUpperCase())).collect(Collectors.toList());
 
+        //include options with allowDetails = true
+        optionMatches.addAll(options.stream().filter(option -> option.isDetailsAllowed()).collect(Collectors.toList()));
```

Comment:
Debated to include allowDetails = true options only when NO matches OR always. These will be at the end of sorted list any way.  Change includes always